source/overview.rst

```diff
@@ -148,7 +165,15 @@ The figure below shows an example WEC-Sim Simulink model for the OSWEC tutorial.
 .. figure:: _images/OSWEC_Model.png
    :width: 200pt
    :align: center       
-   
+
+.. note::
+    Why does this file appear inside the blue box in the workflow diagram? It
+    looks like it's not an input in that diagram, but something to do with the
+    internals. Also, why does is it in a step before the input file is defined 
+    when all the arrows come from the input file into the simulink file? Is 
+    there a relationship between the classes in the input file and the simulink
+    file. That would be good to know.
+
```

Comment:
@H0R5E Can you submit a PR with suggested revisions?

Reply:
Do you have the original file for the workflow diagram?